README.md

```diff
@@ -2,12 +2,21 @@
 
 # `svc-scheduler`
 
+![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/Arrow-air/svc-scheduler?include_prereleases)
+![Sanity Checks](https://github.com/arrow-air/svc-scheduler/actions/workflows/sanity_checks.yml/badge.svg?branch=main)
 ![Rust
 Checks](https://github.com/arrow-air/svc-scheduler/actions/workflows/rust_ci.yml/badge.svg?branch=main)
 ![Python Flake8](https://github.com/arrow-air/svc-scheduler/actions/workflows/python_ci.yml/badge.svg?branch=main)
```

Comment:
Change it to Pep8

Reply:
https://github.com/Arrow-air/svc-scheduler/pull/36/commits/cd81b978e79180ca872b794a32e7fb03f7ff3663